apps/web/src/app/(app)/page.tsx

```diff
@@ -1,36 +1,31 @@
 import React from 'react';
 import { notFound } from 'next/navigation';
+import first from 'lodash/first';
 
-import Button from 'ui/components/Button';
-
-import getPayloadClient from '@web/payload/payloadClient';
-
-async function pageApi() {
-  const payload = await getPayloadClient();
-  try {
-    return await payload.find({
-      collection: 'users'
-    });
-  } catch (err) {
-    return { error: err };
-  }
-}
+import RenderBlocks from '@web/components/RenderBlocks';
+import fetchPayloadData from '@web/lib/fetchPayloadData';
+import { Page } from '@web/payload/payload-types';
 
+// const pageApi = async (slug: string) => {
 export default async function Page() {
-  const data = await pageApi();
+  // const data = (await pageApi('home')) as { docs: Page[] };
+  const data = (await fetchPayloadData((client) =>
+    client.find({
+      collection: 'page',
+      where: {
+        slug: {
+          equals: 'home'
+        }
+      }
+    })
+  )) as { docs: Page[] };
+
+  const page = first(data?.docs);
 
   // if there's an error fetching data, 404
   if ('error' in data) {
     return notFound();
   }
```

Comment:
This doesn't quite work since you're actually concerned about finding `page`, not `data`